### PR TITLE
avoid init irrelevant cases

### PR DESCRIPTION
**Pull request by Richard-Yunchao**
PTAL. Thanks!

### CODE REVIEW

**Comment by kenrussell** [comment]
Is this correct? How is this argument interpreted?

**Comment by kenrussell** [comment]
+ @zhenyao

File: sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js [hunk 1]
```diff
@@ -5253,69 +5253,67 @@ var setParentClass = function(child, parent) {
     /**
      * @return {glsBuiltinPrecisionTests.CaseFactories}
```
Comment (zhenyao) [inline sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js]: Please update the documentation for caseId here.

File: sdk/tests/deqp/functional/gles3/es3fBuiltinPrecisionTests.js [hunk 2]
```diff
@@ -37,12 +37,12 @@ goog.scope(function() {
      * @param {*} context
      * @return {tcuTestCase.DeqpTest}
```
Comment (zhenyao) [inline sdk/tests/deqp/functional/gles3/es3fBuiltinPrecisionTests.js]: Please update the documentation for caseId here.

**Comment by zhenyao** [comment]
Mostly looks good.  Could you please update the two documentation places?

**Comment by Richard-Yunchao** [comment]
Thanks for your review, @kenrussell and @zhenyao . The code has updated accordingly, please have another look. 
Each builtin-precision file (builtinprecision00.html to builtinprecision55.html) test one builtin function, say min, max, mod, ceil, clamp and so forth. The original code will init all builtin functions before the test (but it only test one function). Then uses 'range' to filter irrelverant cases. It is complex and time-consuming. Now each builtin-precision test file only initiate the case it tests. It is simple, direct and clear now: builtinprecision*.html (caseId) -> es3fBuiltinPrecision.js (caseId) -> glsBuiltinPrecision.js (caseId).

**Comment by zhenyao** [comment]
lgtm

**Pull request merged**